content/fr/event-streaming.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Diffusion d'événements en continu (Event Streaming)
+status: Completed
+category: concept
+tags: ["méthodologie", "réseau", ""]
+---
+
+La diffusion d'événements en continu (Event Streaming) est une approche où un logiciel envoie des données d'événements d'une application à une autre pour communiquer en continu ce qu'elles font.
+Imaginez un service diffusant tout ce qu'il fait à tous les autres services.
+Chaque activité effectuée par un service est appelée un événement, d'où le terme de diffusion d'événements en continu.
+Par exemple, le NASDAQ reçoit des mises à jour sur les prix des actions et des matières premières chaque seconde.
+Si vous aviez une application surveillant un ensemble spécifique d'actions, vous voudriez recevoir ces informations en quasi temps réel.
+Yahoo! Finance fournit une [API](/fr/application-programming-interface/) qui récupère les données du NASDAQ et envoie (ou diffuse en continu) les informations (ou événements) de leur application à toute application qui s'y abonne.
+Les données envoyées ainsi que les changements de ces données (prix des actions) sont les événements tandis que le processus de les livrer à une application est la diffusion d'événements.
+
+## Problème auquel il répond
+
+Traditionnellement, Yahoo! Finance mettrait en œuvre des requêtes TCP individuelles.
+Cela serait très inefficace car cela nécessiterait qu'une connexion soit créée pour chaque événement.
+À mesure que les données deviennent de plus en plus temps réel par nature, mettre à l'échelle une telle solution devient inefficace.
```

Comment:
Le "par nature" est bizarre. Je ne sais que dire, le fait de ne pas le mettre ne change pas le sens de la phrase, je pense.
Er les autres, qu'en pensez-vous ?

Reply:
Complètement d'accord avec toi, le "par nature" m'a aussi beaucoup gêné. Je l'ai laissé vis-à-vis de la VO, mais je suis partisan de l'enlever également.
Je le laisse encore, avant d'avoir l'avis de nos collègues.

Reply:
+1

Reply:
fix in https://github.com/cncf/glossary/pull/2950/commits/e2787b396c926257ff0c675b5014a261663ea2e9